pages/icons.js

```diff
@@ -0,0 +1,76 @@
+import Head from "next/head";
+import { useState } from "react";
+import * as FaIcons from "react-icons/fa";
+import * as MdIcons from "react-icons/md";
+import IconCard from "../components/IconCard";
+
+export default function Icons() {
+  const [searchedIconNames, setSearchedIconNames] = useState([]);
+  const [notFound, setNotFound] = useState();
+
+  /**
+   creating an icons object to map the original name of the icon as value
+   the icons name are in camelCase so converting it into lowercase for searchig
+   after searching we need the camelcase string to get the icon from react-icons
+   eg: icons = { mdoutlinetoggleon: 'MdOutlineToggleOn'}
+ */
+
+  const icons = {};
+
+  Object.keys(FaIcons).forEach((key) => {
+    icons[key.toLocaleLowerCase()] = key;
+  });
+
+  Object.keys(MdIcons).forEach((key) => {
+    icons[key.toLocaleLowerCase()] = key;
+  });
+
+  const searchIcons = (value) => {
+    setSearchedIconNames([]);
+    if (value.length < 3) {
+      return setNotFound("Please enter at least 3 characters to search...");
+    }
+    const filteredIconNames = Object.keys(icons)
+      .filter((icon) => icon.includes(value.toLocaleLowerCase()))
+      .map((iconName) => icons[iconName]);
+
+    setSearchedIconNames(filteredIconNames);
+
+    if (!filteredIconNames.length) {
+      return setNotFound("No Icons found");
+    }
+
+    setNotFound();
+  };
+
+  return (
+    <>
+      <Head>
+        <title>LinkFree Search Icons</title>
+        <meta name="description" content="Search LinkFree icon directory" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="flex flex-col px-20 pt-10 align-center">
```

Comment:
The page width is not the same size as other pages https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/search.js#L89

Reply:
Added the page component